Tests/Unit/Controller/Admin/Order/OrderTabDescriptorTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+use Wirecard\Oxid\Controller\Admin\Order\OrderTabDescriptor;
+use Wirecard\Oxid\Tests\Unit\Controller\Admin\TestDataHelper;
+
+class OrderTabDescriptorTest extends Wirecard\Test\WdUnitTestCase
+{
+    /**
+     * @var OrderTab
+     */
+    private $_orderTab;
+
+    protected function dbData()
+    {
+        return TestDataHelper::getDemoData();
+    }
+
+    public function testRender()
+    {
+        $_GET['oxid'] = 'oxid 1';
```

Comment:
Please use `$this->setRequestParameter()`. It also works with GET parameters: https://github.com/OXID-eSales/oxideshop_ce/blob/master/source/Core/Request.php#L26

Reply:
done!